sdk/storage/storage-blob/README.md

```diff
@@ -93,18 +91,6 @@ Or
 const AzureStorageBlob = require("@azure/storage-blob");
 ```
 
-### JavaScript bundle
-
-To use the library with JS bundle in the browsers, simply add a script tag to your HTML pages pointing to the downloaded JS bundle file(s):
-
-```html
-<script src="https://mydomain/azure-storage-blob.min.js"></script>
-```
-
-The JS bundled file is compatible with [UMD](https://github.com/umdjs/umd) standard, if no module system found, following global variable(s) will be exported:
-
-- `azblob`
-
 #### Download
 
 Download latest released JS bundles from links in the [GitHub release page](https://github.com/Azure/azure-sdk-for-js/releases).
```

Comment:
This section was removed in this PR https://github.com/Azure/azure-sdk-for-js/pull/5813.
Why is it still showing up ?

Reply:
I haven't rebased yet (that change was just merged 😁)